app/src/main/java/swati4star/createpdf/fragment/ViewFilesFragment.java

```diff
@@ -4,6 +4,7 @@
 import android.app.Activity;
 import android.content.Context;
 import android.content.DialogInterface;
+import android.nfc.Tag;
```

Comment:
What is the need of this import?

Reply:
Oh sorry, I'll remove it. It was to log errors